token-metadata/program/tests/update_metadata_account_v2.rs

```diff
@@ -667,6 +667,7 @@ mod update_metadata_account_v2 {
         let creator2 = Keypair::new();
         let creator3 = Keypair::new();
         let creator4 = Keypair::new();
+        let creator5 = Keypair::new();
```

Comment:
This is relevant for the testing.  Before this change to the contract, this test will fail with `MustBeOneOfCreatorsError` because update authority was not one of the creators.  My understanding was this is considered a bug so this refactor removes that limitation.